[TASK] Move `screenshots.json` to subfolder `Documentation/`

alexander-nitsche opened the pull request:
The `screenshots.json` files are now found at any folder depth in `public/t3docs`.

Before:
- public/t3docs/TYPO3CMS-Book-ExtbaseFluid/screenshots.json => found
- public/t3docs/extension_builder/Documentation/screenshots.json => **not found**
- public/t3docs/typo3/typo3/sysext/impexp/Documentation/screenshots.json => **not found**

Now:
- public/t3docs/TYPO3CMS-Book-ExtbaseFluid/screenshots.json => found
- public/t3docs/extension_builder/Documentation/screenshots.json => found
- public/t3docs/typo3/typo3/sysext/impexp/Documentation/screenshots.json => found

Fixes: #220

linawolf reviewed (changes requested):
`ddev make-screenshots` works fine. however neither `ddev make-screenshots -t TYPO3CMS-Tutorial-Typoscript45Minutes` nor `ddev make-screenshots -t TYPO3CMS-Tutorial-Typoscript45Minutes/Documentations` finds a screenshot.json in TYPO3CMS-Tutorial-Typoscript45Minutes/Documentation

Edit: the link should have been  `ddev make-screenshots -t TYPO3CMS-Tutorial-Typoscript45Minutes/Documentation` (without s), this one works.

alexander-nitsche commented:
Yes, so you propose to add the feature of taking screenshots of specific folders and its subfolders. I think this feature is a good idea. I added the idea to the docs and adjusted the findAll() and findByPath() to remove redundancy.

alexander-nitsche commented:
If you approve the changes, i will merge the latest two commits into `[FEATURE] Search for `screenshots.json` in any folder depth` before merging the PR.

linawolf commented:
Looks fine to me and works well
